lib/code-path-analysis/debug-helpers.js

```diff
@@ -65,8 +65,8 @@ module.exports = {
         }
 
         debug(
-            state.currentSegments.map(getId).join(",") + ") " +
-            node.type + (leaving ? ":exit" : "")
+            `${state.currentSegments.map(getId).join(",")}) ${
+            node.type}${leaving ? ":exit" : ""}`
```

Comment:
Is this maybe worth converting to an array join? Maybe something like:

```js
debug([
    `state.currentSegments.map(getId).join(","))`,
    `${node.type}${leaving ? ":exit" : ""}`
].join(" "));
```

Also, I'm wondering, is that an unmatched right parenthesis near the end of the first line? (Ping @mysticatea)